tox.ini

```diff
@@ -132,10 +133,10 @@ commands = {toxinidir}/tools/clean.sh {toxinidir}/falcon
 
 [with-cython]
 deps = -r{toxinidir}/requirements/tests
-       cython
 setenv =
     PIP_CONFIG_FILE={toxinidir}/pip.conf
     FALCON_CYTHON=Y
```

Comment:
Maybe there is potential to merge these two envvars into one? Or do they carry a different meaning?

Reply:
I don't know what this variable is. Also i think it's more appropriate to disable cython in our case, so the variable should have disable (or similar) in the name

Reply:
I've searched and `FALCON_CYTHON` is not used anywhere. I'll remove it